Jenkinsfile

```diff
@@ -60,18 +60,19 @@ node('vetsgov-general-purpose') {
       sh "git fetch --no-tags"
 
       def changedFiles = sh(returnStdout: true, script: "git diff --name-only origin/master..origin/${env.BRANCH_NAME}")
-      def homepageChanged = changedFiles.indexOf('fragments/home') > -1
+      def homepageChanged = changedFiles.indexOf('fragments/home/banner.yml') > -1 || changedFiles.indexOf('fragments/home/news.yml') > -1
 
       if (homepageChanged) {
+        def pr = getPullRequest()
         def message = """\
 @channel \
 Pull request opened containing changes to the VA.gov homepage! \
 These changes usually contain content that is high priority, such as for a weather alert or government shutdown. \
 Please review, merge, and if necessary, deploy this change as soon as possible. \
-https://www.github.com/${GITHUB_ORG}/${CONTENT_REPO}/pulls
+https://www.github.com/${GITHUB_ORG}/${CONTENT_REPO}/pull/${pr.getNumber()}
```

Comment:
Something about this `pr.getNumber()` causes this error in Jenkins -

```
Slack Send Pipeline step running, values are - baseUrl: <empty>, teamDomain: dsva, channel: enrique-test, color: #DDDD00, botUser: false, tokenCredentialId: slack-token

org.kohsuke.github.GHPullRequest
```

Any ideas? @omgitsbillryan @wunderhund @wyattwalter

Reply:
I'm far from understanding the deep magic of Jenkins but try something like:
```
def getPullRequestNumber() {
  def github = GitHub.connect()
  def repo = github.getRepository("${GITHUB_ORG}/${CONTENT_REPO}")
  return repo.queryPullRequests().head("${GITHUB_ORG}:${env.BRANCH_NAME}").list().asList().get(0).getNumber()
}
```
I tested a little bit and was getting some serialization errors about the PullRequest object. Something like the above seems to get around that issue by returning a scalar value instead. I dont know.

Reply:
Ah thank you @tskinn! That did it! I don't fully understand why but I changed the function return value to be either a PR object or a number, and that did the trick.